src/tests/test_model.py

```diff
@@ -87,19 +87,27 @@ def test_model_vit_fit(datapipe):
 @pytest.mark.parametrize(
     "litmodule,precision",
     [
-        (CLAYModule, "bf16-mixed" if torch.cuda.is_available() else "32-true"),
-        (ViTLitModule, "bf16-mixed"),
+        (CLAYModule, "16-mixed" if torch.cuda.is_available() else "32-true"),
```

Comment:
Is `bf16-mixed` precision giving any issue while inferencing?

Reply:
It is for the Nvidia Geforce RTX on my local machine, which does not support this type apparently.